solr/solrj/src/java/org/apache/solr/client/solrj/request/QueryRequest.java

```diff
@@ -62,7 +62,7 @@ public String getPath() {
 
   @Override
   protected QueryResponse createResponse(SolrClient client) {
-    return new QueryResponse(client);
+    return new QueryResponse();
```

Comment:
does the fact that this takes in a `client` but does not use it need to be documented?  Just reading it, I would think "thats weried"..

Reply:
No.  Only one place in Solr does it use it, aside from this spot.  So *not* using it is in fact the norm.  IMO it shouldn't even be on the `createResponse` API contract in the first place.  My other PR created yesterday makes that change.